### PR TITLE
Fix typographic errors in docstrings

### PR DESCRIPTION
**Pull request by createyourpersonalaccount**

### CODE REVIEW

File: examples/low-level/old/ulib/IntLib.fst [hunk 1]
```diff
@@ -17,7 +17,7 @@ module IntLib
 
 open Axioms
 
-(** Mathematical useful fonctions **)
```
Comment (R1kM) [inline examples/low-level/old/ulib/IntLib.fst:20]: I think this change loses a bit of the original sense. Maybe "Useful functions for mathematical integers" ?

File: examples/termination/termination.fst [hunk 2]
```diff
@@ -120,7 +120,7 @@ let rec treeMap #a #b f v = match v with
   | Node l ->
     (* NS: this next call seems to be unavoidable. We need to move the refinement "inside" the list.
            An alternative would be to give map a different type accounting for this "outside" refinement.
-           But, it's seeems nicer to give map its normal type *)
+           But, it's seems nicer to give map its normal type *)
```
Comment (R1kM) [inline examples/termination/termination.fst:123]: "it seems" ?

File: ulib/experimental/Steel.PCMReference.fsti [hunk 3]
```diff
@@ -70,7 +70,7 @@ val alloc (#a:Type)
           (requires fun _ -> compatible pcm x x /\ pcm.refine x)
           (ensures fun _ _ _ -> True)
 
-/// Frees reference [r], as long as we have exclusing ownership of [r]
+/// Frees reference [r], as long as we have excluding ownership of [r]
```
Comment (R1kM) [inline ulib/experimental/Steel.PCMReference.fsti:73]: Should be "exclusive"

**Review by R1kM — CHANGES_REQUESTED** [review]
Thanks! I left a few very minor comments, but otherwise this looks good to me, all changes are restricted to fixing typos in comment and do not impact any actual code.

File: examples/low-level/old/ulib/IntLib.fst (hunk 1 above)
Comment (createyourpersonalaccount) [reply examples/low-level/old/ulib/IntLib.fst:20]: Seems like bikeshedding...

File: examples/termination/termination.fst (hunk 2 above)
Comment (createyourpersonalaccount) [reply examples/termination/termination.fst:123]: Good catch!

File: ulib/experimental/Steel.PCMReference.fsti (hunk 3 above)
Comment (createyourpersonalaccount) [reply ulib/experimental/Steel.PCMReference.fsti:73]: Nice!

**Comment by createyourpersonalaccount** [comment]
@R1kM I will update the PR in a couple of days.

**Comment by R1kM** [comment]
Thanks!

**Pull request merged**